CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Changed`
 
 - Suffix used to identify unique fastq pairs from "\_T" to "\_LNUMBER" [#638](https://github.com/nf-core/raredisease/pull/638)
+- Change the regions used by deepavariant for WES samples from bait interval files to target bed files [#636](https://github.com/nf-core/raredisease/pull/636)
```

Comment:
Shouldn't the changelog just list the changes from version 2.2.0 to the next? This is more a fix of PR #633

Reply:
Ah true.. I will remove it.